packages/manager/apps/pci-kubernetes/src/api/data/kubernetes.ts

```diff
@@ -132,3 +132,34 @@ export const resetCluster = async (
   );
   return data;
 };
+
+export const addOidcProvider = async (
+  projectId: string,
+  kubeId: string,
+  params: Partial<TOidcProvider>,
```

Comment:
For a POST use, you need the whole object props, so you should remove the Partial